test/table.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import Table from '../src/client/Table';
+
+describe('<Table />', () => {
+  describe('When users connect to the website', () => {
+    it('table header should display', () => {
+      const wrapper = shallow(<Table />);
+      const header = wrapper.find('thead');
+      expect(header.length).toEqual(1);
+  
+      const headerColumns = header.find('th').map(col => col.text());
+      expect(headerColumns.length).toEqual(6);
+      expect(headerColumns[0]).toEqual('seq');
+      expect(headerColumns[1].length).toEqual(6);
+      expect(headerColumns[headerColumns.length-1]).toEqual('Priority');
+    });
```

Comment:
Not only `head` but also `body` should be tested. 
```js
describe('WHEN GIVEN tableData', () => {
  it('should render counts of rows correctly')
})
```

Also, I suggest to prepare a another set of table data for test and it could be reusable. e.g.
 
`test/mockData.js`
```js
export const mockData = {
  ...
};
```